lib/internal/streams/readable.js

```diff
@@ -486,7 +519,7 @@ Readable.prototype.read = function(n) {
   // 3. Actually pull the requested chunks out of the buffer and return.
 
   // if we need a readable event, then we need to do some reading.
-  let doRead = state.needReadable;
+  let doRead = (state.state & kNeedReadable) !== 0;
```

Comment:
```suggestion
  let doRead = state.state & kNeedReadable;
```

Easier to write and read?

Reply:
I don't want to keep it a number I want to go straight to Boolean